### ISSUE TITLE
module 'imgui' has no attribute 'create_context'

### ISSUE THREAD

**Issue by redgis** [swistakm/pyimgui#118]
Hello, I'm having this strange issue where create_context seem to be missing, testing the example source code : "integrations_glfw3.py"

```
$ python
Python 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 16:02:32) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

```
$ pip install imgui[full]
Collecting imgui[full]
  Downloading https://files.pythonhosted.org/packages/ae/de/89cada9bd84843bf289dfb45e64a8d9c3bae8f0b2e75cdd2d534d27a115a/imgui-0.1.0-cp35-cp35m-win_amd64.whl (777kB)
     |████████████████████████████████| 778kB 6.8MB/s
Collecting pygame; extra == "full" (from imgui[full])
  Downloading https://files.pythonhosted.org/packages/45/a5/580578790aa2f8f274855d45b4c89ffb910f1c72ddb754873b2ae8d7fc7f/pygame-1.9.6-cp35-cp35m-win_amd64.whl (4.3MB)
     |████████████████████████████████| 4.3MB ...
Collecting cocos2d; extra == "full" (from imgui[full])
  Using cached https://files.pythonhosted.org/packages/26/9b/c4985afa0d4ed56a468d9cbf658b8f97514c7bed31d0afe1f7e5dfe38b13/cocos2d-0.6.5.tar.gz
Collecting Cython<=0.28.2,>=0.24; extra == "full" (from imgui[full])
  Downloading https://files.pythonhosted.org/packages/14/63/594da894d3825390e15c8fad53e1c7a68b8a6e8cbddcea028252f3a039f8/Cython-0.28.2-cp35-cp35m-win_amd64.whl (2.4MB)
     |████████████████████████████████| 2.4MB ...
Collecting PySDL2; extra == "full" (from imgui[full])
  Using cached https://files.pythonhosted.org/packages/eb/08/8a81128ff6ca89019de9771425638bc197a28f40c1e377f03626baa5d729/PySDL2-0.9.6.tar.gz
Collecting glfw; extra == "full" (from imgui[full])
  Downloading https://files.pythonhosted.org/packages/a6/66/586a5c0f234128a0e1eb8ee959448f06fe34521cfaaff0cd19b859e55049/glfw-1.8.0-py2.py3-none-win_amd64.whl (63kB)
     |████████████████████████████████| 71kB 4.8MB/s
Collecting PyOpenGL; extra == "full" (from imgui[full])
  Using cached https://files.pythonhosted.org/packages/9c/1d/4544708aaa89f26c97cc09450bb333a23724a320923e74d73e028b3560f9/PyOpenGL-3.1.0.tar.gz
Collecting six>=1.4 (from cocos2d; extra == "full"->imgui[full])
  Using cached https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl
Collecting pyglet>=1.2 (from cocos2d; extra == "full"->imgui[full])
  Using cached https://files.pythonhosted.org/packages/1c/fc/dad5eaaab68f0c21e2f906a94ddb98175662cc5a654eee404d59554ce0fa/pyglet-1.3.2-py2.py3-none-any.whl
Collecting future (from pyglet>=1.2->cocos2d; extra == "full"->imgui[full])
  Using cached https://files.pythonhosted.org/packages/90/52/e20466b85000a181e1e144fd8305caf2cf475e2f9674e797b222f8105f5f/future-0.17.1.tar.gz
Building wheels for collected packages: cocos2d, PySDL2, PyOpenGL, future
  Building wheel for cocos2d (setup.py) ... done
  Stored in directory: C:\Users\Regis\AppData\Local\pip\Cache\wheels\52\c3\70\37a1539ad4fefa156f4b8552fe8b7184f30a2a6b18c2c44852
  Building wheel for PySDL2 (setup.py) ... done
  Stored in directory: C:\Users\Regis\AppData\Local\pip\Cache\wheels\fd\fe\9b\dae62e373920a057fb8e046b304ff39fcf5edce9ff9eac547d
  Building wheel for PyOpenGL (setup.py) ... done
  Stored in directory: C:\Users\Regis\AppData\Local\pip\Cache\wheels\6c\00\7f\1dd736f380848720ad79a1a1de5272e0d3f79c15a42968fb58
  Building wheel for future (setup.py) ... done
  Stored in directory: C:\Users\Regis\AppData\Local\pip\Cache\wheels\0c\61\d2\d6b7317325828fbb39ee6ad559dbe4664d0896da4721bf379e
Successfully built cocos2d PySDL2 PyOpenGL future
Installing collected packages: pygame, six, future, pyglet, cocos2d, Cython, PySDL2, glfw, PyOpenGL, imgui
Successfully installed Cython-0.28.2 PyOpenGL-3.1.0 PySDL2-0.9.6 cocos2d-0.6.5 future-0.17.1 glfw-1.8.0 imgui-0.1.0 pygame-1.9.6 pyglet-1.3.2 six-1.12.0
```

```
$ python integrations_glfw3.py
[...]\Python-3.5\lib\site-packages\imgui\integrations\__init__.py:15: FutureWarning:
    Integration layer in imgui.integrations is still in experimental mode.
    Expect mayor changes in future versions. For safety it is recommended
    to use code vendoring for specific integration impementation.

    This subpackage should become stable in 1.0.0 version.


  """, FutureWarning
Traceback (most recent call last):
  File "integrations_glfw3.py", line 81, in <module>
    main()
  File "integrations_glfw3.py", line 10, in main
    imgui.create_context()
AttributeError: module 'imgui' has no attribute 'create_context'
```

**Comment by redgis** [comment]
If I comment "imgui.create_context()" and "imgui.end_frame()", things seem to work. Is this related with a recent change that may not be propagated yet in the packages in pip repository ?

**Comment by swistakm** [comment]
Yes, these are due to latest changes that haven't been release yet as a stable version on PyPI.

You can either install a pre-release version (see [installation info](https://github.com/swistakm/pyimgui/releases/tag/1.0.0a2)) or use examples from latest stable [version tag](https://github.com/swistakm/pyimgui/tree/0.1.0).

**Comment by redgis** [comment]
Perfect, thank you !

**Comment by swistakm** [comment]
Released as `1.0.0`
